src/test/java/htsjdk/samtools/cram/structure/CompressionHeaderEncodingMapTest.java

```diff
@@ -53,11 +58,51 @@ public void testAllCRAMEncodingsRoundTripThroughStream() throws IOException {
             final byte[] roundTrippedBytes = baos.toByteArray();
             try (final InputStream is = new ByteArrayInputStream(roundTrippedBytes)) {
                 final CompressionHeaderEncodingMap roundTippedEncodingMap = new CompressionHeaderEncodingMap(is);
-                assertExpectedHTSJDKGeneratedEncodings(roundTippedEncodingMap, Collections.EMPTY_SET);
+                assertExpectedHTSJDKGeneratedEncodings(roundTippedEncodingMap, CompressionHeaderEncodingMap.DATASERIES_NOT_READ_BY_HTSJDK);
             }
         }
     }
 
+    @Test(dataProvider = "readCramTestFiles", description = "make sure that obsolete dataseries, TC and TN are ignored on CRAM read")
```

Comment:
Maybe name this data provider the more specific `obsoleteDataSeriesCRAM` like you did for the other test. In fact, we usually don't create a data provider at all for a one-off test case like this. Either way is fine, but if we keep it, lets rename it.